modules/gapi/src/backends/ocl/goclcore.cpp

```diff
@@ -11,6 +11,24 @@
 #include <opencv2/gapi/ocl/core.hpp>
 #include "backends/ocl/goclcore.hpp"
 
+#ifdef HAVE_DIRECTX
+#ifdef HAVE_D3D11
+#pragma comment(lib,"d3d11.lib")
```

Comment:
Same question about handling it at cmake level...

Reply:
Answered below